src/TransformOutput.js

```diff
@@ -41,7 +41,9 @@ function transform(transformer, transformCode, code) {
     return Promise.resolve(result).then(result => {
       let map = null;
       if (typeof result !== 'string') {
-        map = new SourceMapConsumer(result.map);
+        if (result.map) {
```

Comment:
was getting null for result.map for some reason.. might just be a configuration issue